modules/RouterUtils.js

```diff
@@ -1,26 +1,7 @@
-import deprecateObjectProperties from './deprecateObjectProperties'
-
 export function createRouterObject(history, transitionManager) {
   return {
     ...history,
     setRouteLeaveHook: transitionManager.listenBeforeLeavingRoute,
```

Comment:
Maybe the breakage is just dropping `router.listen` entirely? No one had (easy) access to it to begin with, so it's not a major loss.